config/game_options.txt

```diff
@@ -271,7 +271,7 @@ EVENTS_MIN_PLAYERS_MUL 1
 ## AI ###
 
 ## Allow the AI job to be picked.
-ALLOW_AI
+#ALLOW_AI
```

Comment:
Shouldn't we remove this and it's configuration code all together if we aren't going to have AI?

Reply:
I feel like leaving the config in doesn't hurt and would be easier to keep track of in the future. But there's also no reason it needs to be kept.